consensus/poet/simulator/simulator_rk_pub.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
I went back and forth on whether or not to check this in.  We can discuss what the proper thing to do is as I have the same dilemma with regard to the IAS report public key.  We need to distribute it (so that when bringing up the validator network we can put it on the blockchain in config) and so was going to put it in repo so that we can package it up.  Same goes for the simulator report public key, although not as important as the validator registry TP will fall back to this key anyway.